buildconfig/CMake/SipQtTargetFunctions.cmake

```diff
@@ -73,37 +49,26 @@ function(mtd_add_sip_module)
     endif()
   endforeach()
 
-  # Run sip code generator
-  if(PARSED_PYQT_VERSION EQUAL 4 AND USE_PRIVATE_SIPPYQT4)
-    set(_sip_include_dir ${PYQT4_SIP_INCLUDE_DIR})
-    set(_sip_executable ${PYQT4_SIP_EXECUTABLE})
+  # Configure final module
+  set(_module_spec ${CMAKE_CURRENT_LIST_DIR}/${PARSED_SIP_SRC})
+  if(PARSED_PYQT_VERSION EQUAL 5)
+    if(SIP_BUILD_EXECUTABLE)
+      _add_sip_library(
+        ${PARSED_TARGET_NAME} ${PARSED_MODULE_NAME} ${_module_spec}
+        ${PARSED_PYQT_VERSION} _sip_include_deps
+      )
+    else()
+      _add_sip_library_v4(
```

Comment:
Do we have _sip_library_v4 anywhere, based on the work above I thought SIP was V5/6?

Reply:
Yeah only macOS has v6 at the moment. Everything else, including conda at this point, are sip v5 so we need to keep this for now unfortunately.